TOOLS/dylib-unhell.py

```diff
@@ -89,8 +89,16 @@ def process_libraries(libs_dict, binary, processed = []):
 def process_swift_libraries(binary):
     command = ['xcrun', '--find', 'swift-stdlib-tool']
     swiftStdlibTool = subprocess.check_output(command, universal_newlines=True).strip()
+    #from xcode11 the dynamic swift libs reside in a seperate directory from
```

Comment:
`separate`. Also I think the Python comment style has a space after the hash symbol?